crates/sui-framework/packages/dwallet-system/sources/ethereum_state.move

```diff
@@ -82,6 +80,7 @@ module dwallet_system::ethereum_state {
         optimistic_update_bytes: vector<u8>,
         latest_ethereum_state: &mut LatestEthereumState,
         eth_state: &EthereumState,
+        is_init_state: bool,
```

Comment:
Please explain this channe. This is a public function, meaning that the user can freely specify this value at will. What is its purpose? what will happen if a user says its an init state when its not?

Reply:
document this

Reply:
is_init_state is not a very good name, ill change it.
This is a flag to figure how to apply the updates.
If it is not the first time we apply the update(which only happens the first time we use and update the `LatestEthereumState`), then we must only verify and apply the finality update & optimistic update, and only if the "regular" update has a new sync committee - we would verify and apply it as well(otherwise we do not use it).

If you put the wrong value in this flag - the state verification would fail.